### PR TITLE
Add wearable app description

### PR DESCRIPTION
**Pull request by pengyuan-zhou**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +209,15 @@ jupyter notebook
 - Get the IP address of your computer by running `ip address show | grep "inet " | grep -v 127.0.0.1` if using Linux/Mac. For windows there are different steps. Alternatively, if you want to run the demo app in the emulator, use `10.0.2.2` as the IP address.
 - Use this IP address and the port (default:5000) in your login screen to supply the PyGrid server url, e.g., 10.0.2.2:5000
 
+### Try the wearable app
+
```
Comment (vkkhare) [inline README.md:213]: The title should be something generic for example,

`Built on KotlinSyft`

File: README.md [hunk 2]
```diff
@@ -209,6 +209,15 @@ jupyter notebook
...
+<img src="project_resources/watch.png " width="200" height="200" />
+
+[Federated-Wearables](https://github.com/FL-Wearable/Trainer) is a demo app for cross-device federated learning over wearables. The smartwatch app offloads the collected data to the paired phone app via Bluetooth for training in FL, and gets the updated model to run interface in real time.
```
Comment (vkkhare) [inline README.md:216]: I think one line or two should be good enough. People will eventually open the link to read about the app.

**Review by vkkhare — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pengyuan-zhou) [reply README.md:213]: cool thanks

File: README.md (hunk 2 above)
Comment (pengyuan-zhou) [reply README.md:216]: ok

**Review by vkkhare — APPROVED** [review]

**Pull request merged**